x-pack/plugins/synthetics/common/constants/rest_api.ts

```diff
@@ -48,6 +48,7 @@ export enum API_URLS {
   SYNTHETICS_HAS_ZIP_URL_MONITORS = '/internal/uptime/fleet/has_zip_url_monitors',
 
   // Project monitor public endpoint
-  SYNTHETICS_MONITORS_PROJECT_LEGACY = '/api/synthetics/service/project/monitors',
   SYNTHETICS_MONITORS_PROJECT = '/api/synthetics/project/{projectName}/monitors',
+  SYNTHETICS_MONITORS_PROJECT_DELETE = '/api/synthetics/project/{projectName}/monitors/_delete',
```

Comment:
i don't think for this seprate route is needed, METHOD distinction should be enough

Reply:
The AC currently specifies using `POST`, and we already have a POST route. This draft PR directly follows the AC, but there is discussion if we should use `DELETE` route.